schemas/IDLs/org.gel.models.report.avro/6.2.0/CommonInterpreted.avdl

```diff
@@ -252,6 +252,28 @@ enum TimeUnit {years, months, weeks, days, hours, minutes, na}
     */
     enum AmpClinicalOrExperimentalEvidenceLevel{levelA, levelB, levelC, levelD}
 
+    record User{
+        string username;
+        string role;
+        string email;
+        array<string> groups;
+    }
+
+    record UserComment{
+        /**
+        Comment text
+        */
+        string comment;
+        /**
+        User who created comment
+        */
+        union {null, User} user;
+        /**
+        Time comment was created (epoch timestamp in milliseconds e.g. 1605561554148)
```

Comment:
Can it be ISO 8601?

https://xkcd.com/1179/

Reply:
@gerlv I used the timestamp in milliseconds because there was an avro type available for it (http://avro.apache.org/docs/1.8.0/spec.html#Timestamp+%2528millisecond+precision%2529), so it will fail validation if not a valid timestamp.
I can't see an avro type for ISO 8601 datetime, although could just have it as a string field and do validation in cipapi if you think that would be better?

Reply:
yes please, can we use ISO 8601 full date and validate in CIPAPI?

Reply:
done